sdk/js-query/src/query/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./request";
+export * from "./response";
+export * from "./utils";
+export * from "./ethCall";
+export * from "./consts";
```

Comment:
Is there any reason someone would want to use BinaryWriter outside of this directory?

Reply:
I don't think so, I think we should hide the implementation details